creator-node/src/index.ts

```diff
@@ -270,7 +270,7 @@ const startApp = async () => {
     // NOTE: log messages emitted here may be swallowed up if using the bunyan CLI (used by
     // default in `npm start` command). To see messages emitted after a kill signal, do not
     // use the bunyan CLI.
-    logger.info('Shutting down db and express app...', signal, error)
+    logger.error('Shutting down db and express app...', signal, error)
     sequelize.close()
     if (appInfo) {
```

Comment:
did you see the error `ReferenceError: Cannot access 'appInfo' before initialization`
idk if worth fixing really but just fyi in case you think it is

Reply:
oh no i didn't let me take a look

Reply:
i posted in that same triage thread

Reply:
i did some digging but i was not able to find any instances of this error in loggly in the last 30 days. it's possible it got introduced here with these changes https://github.com/AudiusProject/audius-protocol/pull/3881/files#diff-fde27aec1d059a20ad60914bf50f20439230a767c4b362288d05aaa9e4b22a40R182. i kinda don't want to touch this for now and rock the boat

Reply:
yeh prob - agree not worth, this only happened on figment's node bc it exited